plugins/modules/system/xfconf.py

```diff
@@ -144,12 +145,11 @@ def __init__(self, module):
         self.method_map = dict(zip((self.SET, self.GET, self.RESET),
                                    (self.set, self.get, self.reset)))
 
-        # @TODO This will not work with non-English translations, but xfconf-query does not return
-        #       distinct result codes for distinct outcomes.
         self.does_not = 'Property "{0}" does not exist on channel "{1}".'.format(self.property, self.channel)
 
         def run(cmd):
-            return module.run_command(cmd, check_rc=False)
+            # Force language to 'C' to ensure return values are always formatted in english, even in non-english environments
+            return module.run_command(cmd, check_rc=False, environ_update=dict(LANG='C'))
```

Comment:
I wrote that @TODO comment above because the idea occurred to me, but the one time I tried (not thoroughly enough, I'm afraid) to trigger the error, I failed to do so. There's a thousand reasons why that could have happened, but without a formal test-suite these things are more guesswork than real facts.

Reply:
I've run into the non-existent property error myself and this module was unable to detect/process it correctly because I'm on a dutch locale. So I started off with adding multiple variants of the string and checking against all of them, before digging a bit into xfconf-query and finding out that passing the LANG envvar supposedly makes it always use english.

Reply:
Awesome, great that you were able to validate that suspicion! Please see @felixfontein comment about setting the variable for the entire module (instead of specifically at `run_command`). And please add the changelog fragment.